qa-tests-backend/src/main/groovy/orchestratormanager/Kubernetes.groovy

```diff
@@ -2323,6 +2323,7 @@ class Kubernetes implements OrchestratorMain {
         Container container = new Container(
                 name: deployment.containerName ? deployment.containerName : deployment.name,
                 image: deployment.image,
+                imagePullPolicy: "IfNotPresent",
```

Comment:
@porridge Since we have image prefetcher can we set this to never so if image is not prefetched it will fail immediately?